helpers/timeduration_test.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestMarshalTimeDuration(t *testing.T) {
```

Comment:
Wouldn't hurt to make a marshal/unmarshal test for the entire action (and all other actions), we should probably add that as a task to be done.